docs/t-sql/database-console-commands/dbcc-checktable-transact-sql.md

```diff
@@ -174,7 +174,11 @@ By default, DBCC CHECKTABLE performs parallel checking of objects. The degree of
 Parallel checking can be disabled by using trace flag 2528. For more information, see [Trace Flags &#40;Transact-SQL&#41;](../../t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md).
     
 > [!NOTE]    
-> During a DBCC CHECKTABLE operation, the bytes that are stored in a byte-ordered user-defined type column must be equal to the computed serialization of the user-defined type value. If this is not true, the DBCC CHECKTABLE routine will report a consistency error.    
+> During a DBCC CHECKTABLE operation, the bytes that are stored in a byte-ordered user-defined type column must be equal to the computed serialization of the user-defined type value. If this is not true, the DBCC CHECKTABLE routine will report a consistency error. 
+
+> [!NOTE]
+> This feature is not available in every edition of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)]. For more information, see parallel consistency check in the RDBMS Manageability section of [Features Supported by the Editions of SQL Server](../../sql-server/editions-and-components-of-sql-server-version-15.md). 

```

Comment:
@rrg92 Please change your link to:
`../../sql-server/editions-and-components-of-sql-server-version-15.md#RDBMSM`

@rothja after this please merge.

Reply:
I updated the link directly because editing was allowed. I will merge. Thank you @pmasl and @rrg92!